### ISSUE TITLE
Elk not working

### ISSUE THREAD

**Issue by schehlmj** [mjbvz/vscode-markdown-mermaid#287]
This is not rendering in elk as it should be:
<pre>
```mermaid
%%{init: {"flowchart": {"defaultRenderer": "elk"}} }%%
flowchart TD
    C{Let me think}
    C -->|One| D[Laptop]
    C -->|Two| E[iPhone]
    C -->|Three| F[fa:fa-car Car]
```
</pre>

The mermaid.live editor works as expected:

![Mermaid-Elk-issue](https://github.com/user-attachments/assets/6230b8dc-9d47-4d8e-b462-e21343a6cb80)

**Comment by worbunkel** [comment]
PR #288 fixes this

**Comment by mjbvz** [comment]
Fixed by #288
